product-service/functions/createProduct.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+const AWS = require("aws-sdk");
+const db = new AWS.DynamoDB.DocumentClient();
+
+const productTable = process.env.PRODUCT_TABLE;
+
+module.exports.handler = async (event) => {
```

Comment:
Here we are only setting the product itself but not the stock, and we should, since without this this product will never be returned in the GET endpoint